CONTRIBUTING.md

```diff
@@ -85,100 +86,62 @@ We use [Plop](https://plopjs.com/) to scaffold new components rather than copyin
 The configuration allows you to create new core and package components.
 Follow the steps below to create a new component:
 
-1. Run `npm run create-component` and follow the prompts
-2. Run `npm run prepare` to tell Lerna to bootstrap your new component folder
+1. Run `rush create-component` and follow the prompts
```

Comment:
My PR will simplify this command to `rush create`, then the prompt asks "component" or "package".